### PR TITLE
viable 1b12 (new cask)

### PR DESCRIPTION
**Pull request by mariolopjr**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

I tried running `brew audit` and `brew style` but both did not like paths and this is a new cask so not sure how to run it correctly.

### CODE REVIEW

File: Casks/v/viable.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    end
+  end
+
+  depends_on macos: ">= :monterey"
+
+  app "viable#{version.csv.first}/Viable.app"
+
+  zap trash: [
+    "~/Library/Preferences/co.eclecticlight.Viable.plist",
+  ]
```
Comment (daeho-ro) [inline Casks/v/viable.rb:27]:
```suggestion
  zap trash: "~/Library/Preferences/co.eclecticlight.Viable.plist"
```

File: Casks/v/viable.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+cask "viable" do
+  version "1b12,2023.11"
+  sha256 "6613cf7f0ae9a6737eeec8a15f3110e53cb6986b51f3a236913046e8d3769d00"
+
+  url "https://eclecticlightdotcom.files.wordpress.com/#{version.csv.second.major}/#{version.csv.second.minor}/viable#{version.csv.first}.zip",
+      verified: "eclecticlightdotcom.files.wordpress.com/"
```
Comment (daeho-ro) [inline Casks/v/viable.rb:6]:
```suggestion
  url "https://eclecticlight.co/wp-content/uploads/#{version.csv.second.major}/#{version.csv.second.minor}/viable#{version.csv.first}.zip"
```

File: Casks/v/viable.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  name "viable"
+  desc "Create and run macOS virtual machines on Apple silicon Macs"
+  homepage "https://eclecticlight.co/virtualisation-on-apple-silicon"
+
+  livecheck do
+    url :homepage
+    regex(%r{href=.*?/(\d+)/(\d+)/viable[._-]?v?(\d+(?:\.\d+)*\w?)\.zip}i)
```
Comment (daeho-ro) [inline Casks/v/viable.rb:13]:
```suggestion
    regex(%r{href=.*/(\d+)/(\d+)/viable[._-]?v?(\d+\w?(?:[._-]?\d+))\.zip}i)
```

File: Casks/v/viable.rb (hunk 2 above)
Comment (bevanjkay) [reply Casks/v/viable.rb:6]: The

**Comment by krehel** [comment]
Livecheck regex is not ideal as this appears to be a beta and contains a letter, hopefully this stabilizes in the future to a proper version number.  Right now it's a bit generous, but anchored between text so hopefully avoids false positives

**Review by razvanazamfirei — APPROVED** [review]

**Comment by razvanazamfirei** [comment]
Thanks so much for your first contribution, @mariolopjr! For future reference, we have some instructions on how to add [zap stanzas](https://docs.brew.sh/Cask-Cookbook#zap-creation)! :) 

Without contributions like yours, it'd be impossible to keep our project going. 👍

**Pull request merged**